Add RPN Calculator exercise (#1250)
exercism/rust: exercises/concept/rpn-calculator/.docs/introduction.md

> # Introduction

Reviewer: Probably worth adding a sentence to this section about what a stack is: like a stack of paper, the last items in are the first ones out.

Reviewer 2: FWIW, the [book](https://doc.rust-lang.org/book/ch04-01-what-is-ownership.html#the-stack-and-the-heap) uses the metaphor of plates to discuss the stack. Perhaps we do that here too for consistency?

Author: Should be resolved by 2fae730 & ed5ed0d

> Rust's vector implementation, [`std::vec::Vec`](https://doc.rust-lang.org/std/vec/struct.Vec.html), can be used to simulate a stack by using its `push()` and `pop()` methods.

Reviewer: ```suggestion
Rust's vector implementation, [`std::vec::Vec`](https://doc.rust-lang.org/std/vec/struct.Vec.html), can be used as a stack by using its `push()` and `pop()` methods.
```

It's not really simulating it; it's really a stack.

Author: Fixed in 7f6a9a4
